src/functions-and-arrays.js

```diff
@@ -1,12 +1,34 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else if (num2 > num1) {
+    return num2;
+  } else {
+    return num1; 
+  }
+}
 
 
 
-// Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+  if (arr.length === 0) {
+    return null;
+  }
+
```

Comment:
Here, you have your function working, because you stored the first element of the array in your variable and then you looped through the rest of the array. However, that might not be always the procedure. It is good practices if you start with an empty string and then you run your loop for the whole array.
Here's how you should do it:
```
function findLongestWord(arr) {
  if (arr.length === 0) return null;

  let longestWord = '';
  for (let i = 0; i < arr.length; i++) {
    if (longestWord.length < arr[i].length) {
      longestWord = arr[i];
    }
  }
  return word;
}
```